docs/user/whats-new.asciidoc

```diff
@@ -90,55 +68,47 @@ image::images/metric-customization.png[Customization options for a metric panel]
+These customers are now enjoying higher alerting capacity and faster response times without the need for any additional configuration or hardware. Furthermore, these improvements also bring in a new unparalleled scalability for our largest alerting customers, enabling deployments of up to 192 Kibana nodes running alerting rules.
 
-image::images/ip-location-processor.png[The IP Location processor]
+We are very excited to see how these enhancements empower your alerting strategies with faster, more efficient, and scalable performance.
 
 [discrete]
-==== File uploader PDF support
-The file uploader provides a quick way to upload data and start using Elastic. In 8.16, we are improving it to allow you to upload data from PDF files.
+==== Kibana Cases support for file attachments through API actions 
+The latest enhancements to Kibana Cases introduce highly requested functionality for managing security incidents and workflows. Users can now attach files to Kibana Cases via the public API, allowing them to enrich cases with supporting documentation, evidence, and other critical information. This improvement marks a significant enhancement in case management, providing incident response teams with greater flexibility to include all relevant details directly within their cases.
 
-image:https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blte8f0b295330b7e68/67222fb0ca492a5044b51bd8/file_uploader_pdf.gif[File uploader with PDF support]
+Additionally, this release expands integration capabilities with third-party systems such as ServiceNow and Tines. Users can now programmatically update cases through API calls from external platforms, streamlining workflows and fostering seamless collaboration across tools. These features make it easier to automate case updates and efficiently track complex security incidents in real time. For more details, see the https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-createcasedefaultspace[API documentation].
 
 [discrete]
-=== Developer Tools Console redesign
-We're excited to introduce a number of improvements to the overall user experience on one of our most popular features: **Console**. If you're new to Console, you will be welcomed by an onboarding tour that will help you get started quickly with your first requests. And if you're already a regular Console user, you will notice a variety of new features, including the ability to copy outputs to the clipboard, import and export request files, enjoy improved responsiveness, and other quality of life improvements. 
+==== Kibana Connector now supports Jira Data Center
```

Comment:
```suggestion
==== Jira connector now supports Jira Data Center
```
I will double-check but presume it's this specific connector that's changed.

Reply:
You're right!